### DISCUSSION TITLE
Move Zwave-js-ui HA addon to a stand alone LXC Container

### DISCUSSION THREAD

**Discussion by daschmidt1994** [zwave-js/zwave-js-ui#3499] [Q&A] [answered]
I have moved from my odroid to a Proxmox server.
Now my home assistant is running on a VM.
Zwave-Js-ui is installed as addon in HA, now I will move to zwave-js-ui in a extra lxc container.
Is this possible without lose data/devices?
When yes how?
Does the HA websocket-server work?

**Comment by robertsLando** [comment, marked as answer]
So essentially what you need is only a store backup. In order to do it just go to the store explorer tab and click on the FAB button, select then the action backup, it will download a zip file with all the files needed, you can then do the same but select restore action in order to restore it in another instance. It shouldn't be necessary but you can also do an NVM backup if needed to backup the stick memory, that will not be loosed when changing z-ui instance as it's stored on the stick but it's always safer to have it
